app/assets/stylesheets/components/task_list/task-header-component.scss

```diff
@@ -0,0 +1,7 @@
+.app-task-hint {
+  @extend .govuk-hint;
```

Comment:
Just ran this and it looks like this line isn't needed. Otherwise, everything looks great

Reply:
I put this in to cover if you forgot to add the `<p>` in the locales. - I should have mentioned this in the commit - I know it's ugly, but I am confident we will be back to sort out other visual issues once we have shipped public beta!